test/spec/account/auth.spec.ts

```diff
@@ -1,100 +1,95 @@
 // tslint:disable-next-line: no-var-requires
-require('module-alias/register');
+require('module-alias/register')
 
-import chai from 'chai';
+import chai from 'chai'
 // tslint:disable-next-line: import-name
-import spies from 'chai-spies';
-chai.use(spies);
-import chaiHttp from 'chai-http';
-import { Application } from 'express';
-import { respositoryContext, testAppContext } from '../../mocks/app-context';
+import spies from 'chai-spies'
+chai.use(spies)
+import chaiHttp from 'chai-http'
+import { Application } from 'express'
+import { respositoryContext, testAppContext } from '../../mocks/app-context'
 
-import { AuthHelper } from '@helpers';
-import { App } from '@server';
-import { Account } from '@models';
+import { AuthHelper } from '../../../src/helpers'
+import { App } from '../../../src/server'
+import { Account } from '../../../src/models'
 
-chai.use(chaiHttp);
-const expect = chai.expect;
-let expressApp: Application;
+chai.use(chaiHttp)
+const expect = chai.expect
+let expressApp: Application
 
 before(async () => {
-  await respositoryContext.store.connect();
-  const app = new App(testAppContext);
-  app.initializeMiddlewares();
-  app.initializeControllers();
-  app.initializeErrorHandling();
-  expressApp = app.expressApp;
-});
+  await respositoryContext.store.connect()
+  const app = new App(testAppContext)
+  app.initializeMiddlewares()
+  app.initializeControllers()
+  app.initializeErrorHandling()
+  expressApp = app.expressApp
+})
 
 describe('POST /account', () => {
```

Comment:
Remove all things account related as it seems you have already removed portion of it.